.travis.yml

```diff
@@ -20,15 +20,15 @@ branches:
     - master-last_success
     - release-*
 language: node_js
-node_js: "12"
+node_js: "12.18.2"
 addons:
   apt:
     packages:
       - google-chrome-stable
       - unzip
 before_install:
   - . ./scripts/setEnv.sh
-  - npm install -g npm@5.6.0
+  - npm install -g npm@6.14.5
```

Comment:
We should just remove this and default to the node_js version.

Reply:
removed